enterprise/backend/src/metabase_enterprise/metabot_v3/tools/generate_insights.clj

```diff
@@ -23,4 +23,5 @@
                         .getBytes
                         codecs/bytes->b64-str)
                     id)]
-    {:output (str (public-settings/site-url) "/auto/dashboard/" entity-type "/" entity-id)}))
+    {:output (str (public-settings/site-url) "/auto/dashboard/" entity-type "/" entity-id)
+     :reactions [{:type :metabot.reaction/redirect :url (str "/auto/dashboard/" entity-type "/" entity-id)}]}))
```

Comment:
nit: I suggest extracting and re-using `(str "/auto/dashboard/" entity-type "/" entity-id)`.